src/components/SkillCard/SkillCard.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './SkillCard.scss'
+
+const SkillCard = ({image, skill}) => {
```

Comment:
we could get the bgColor prop here and use inline string concatenation with the skillCard class

...
```<div className={`skillCard bg-${bgColor}`}>```

Reply:
I'll add them with comments.